src/Packagist/WebBundle/Controller/WebController.php

```diff
@@ -48,7 +48,7 @@ public function searchFormAction(Request $req)
         $this->computeSearchQuery($req, $filteredOrderBys);
 
         if ($req->query->has('search_query')) {
-            $form->bind($req);
+            $form->submit($req);
```

Comment:
This is actually also soon to be deprecated.

See https://github.com/symfony/symfony/blob/2.8/src/Symfony/Component/Form/Form.php#L509